app/web/markdown/volunteer/technical-support-analyst.md

```diff
@@ -27,19 +27,20 @@ The TSA will possess the skills to be able to identify the cause(s) of reported
 ### Skills & Experience
 
 An ideal candidate would have the following skills:
+- Extensitve experience with couch surfing, preferably on Couchers.org, CouchsurfingTM and/or BeWelcome;
```

Comment:
™, "or similar". i wouldn't name BW

Reply:
I wanted to name them because they are closest to us in values and community besides CS